src/fuchsia/mod.rs

```diff
@@ -2602,6 +2602,7 @@ pub const PR_SET_MM_MAP: ::c_int = 14;
 pub const PR_SET_MM_MAP_SIZE: ::c_int = 15;
 
 pub const PR_SET_PTRACER: ::c_int = 0x59616d61;
+pub const PR_SET_PTRACER_ANY: ::c_int = 0xffffffffffffffff;
```

Comment:
The type is most likely wrong, too small to carry this value, ought to be `c_ulong`

Reply:
done

Reply:
cool, all tests pass it seems 🙂